examples/flutter_gallery/lib/gallery/drawer.dart

```diff
@@ -214,19 +238,19 @@ class GalleryDrawer extends StatelessWidget {
               children: <TextSpan>[
                 new TextSpan(
                   style: aboutTextStyle,
-                  text: "Flutter is an early-stage, open-source project to help "
-                  "developers build high-performance, high-fidelity, mobile "
-                  "apps for iOS and Android from a single codebase. This "
+                  text: 'Flutter is an early-stage, open-source project to help '
+                  'developers build high-performance, high-fidelity, mobile '
```

Comment:
ideally, indent the string so it's vertically aligned.

Reply:
Done.